README.md

```diff
@@ -40,6 +40,30 @@ Table references and indexes in a migration will be applied to the same tenant p
 Apartmentex.new_tenant(Repo, tenant)
 ```
 
+When you need to update a tenant's schema based on new migrations, you can run:
+
+```elixir
+# Runs all migrations necessary for the tenant, based on that tenant's
+`schema_migrations` table
+# Returns a tuple that is either:
+# {:ok, prefix_of_tenant, versions_migrated}
+# {:error, prefix_of_tenant, db_error_message}
+
+{status, prefix, versions_or_error} = Apartmentex.migrate_tenant(Repo, tenant)
```

Comment:
would you consider rather than having just migrate_tenant and rollback tenant, we could pass direction and :to into the function, similar to how ecto does it? 

Something like:
```elixir
def migrate_tenant(repo, tenant, opts \\ []) do
  ...
end
```
where we can pass in direction: :up, to: 1234456667 in the options? the opts can default to :up, and to: :all, so with nothing else, we just get the expected full migration, but for those with a specific purpose, we can accept options.